lib/tbot/ca_rotation.go

```diff
@@ -156,6 +156,7 @@ func (b *Bot) watchCARotations(ctx context.Context, queueReload func()) error {
 				types.HostCA:     clusterName,
 				types.UserCA:     clusterName,
 				types.DatabaseCA: clusterName,
+				types.OpenSSHCA:  clusterName,
```

Comment:
@timothyb89 We want to introduce a new CA. Where should we add that CA to the tbot? I don't think `tbot` will be using it in any way.

Reply:
Normally they'd be added to [`configtemplate_tls_cas.go`](https://github.com/gravitational/teleport/blob/master/lib/tbot/config/configtemplate_tls_cas.go) if there's any reason clients might want to use the TLS form of the cert, and [`configtemplate_ssh_client.go`](https://github.com/gravitational/teleport/blob/master/lib/tbot/config/configtemplate_ssh_client.go) if there's any SSH client config changes needed, like new `known_hosts` entries.

If these CAs are never really exposed to users we shouldn't need to do anything.

Reply:
@capnspacehook `tbot` doesn't need to know about OpenSSH CA rotation. Please remove it from the rotation watcher here.

Reply:
Done